packages/ui-react/src/components/Favorites/Favorites.tsx

```diff
@@ -53,7 +42,7 @@ const Favorites = ({ playlist, error, isLoading, accessModel, hasSubscription, o
           playlist={playlist}
           onCardHover={onCardHover}
           cols={cols}
-          isLoading={isLoading}
+          isLoading={false}
```

Comment:
Aren't we losing the loading state (placeholder data)?

Reply:
For favorites? They are loaded and awaited during login (see PR decsr).